regression-test/suites/export/test_array_export.groovy

```diff
@@ -174,7 +174,7 @@ suite("test_array_export", "export") {
         result = sql """
             SELECT * FROM ${tableName} t ORDER BY k1 INTO OUTFILE "file://${outFile}/";
         """
-        url = result[0][3]
+        def url = result[0][3]
         urlHost = url.substring(8, url.indexOf("${outFile}"))
```

Comment:
maybe other variables also have the same problem, such as `urlHost` ?

Reply:
urlHost is defined as local variable